### PR TITLE
Added Polish Language support,Updated en_us.json & pl_pl.json

### PR DESCRIPTION
**Pull request by L0um15**
I also edited BookCommand to make "You are not holding a book" translatable to other languages and removed unused import.
There is also a word that i cant translate because of many meanings. In the future i will update it.

### CODE REVIEW

File: src/main/resources/assets/clientcommands/lang/pl_pl.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  "enchCrack.xpSeed.one": "Ziarno XP: %08X",
+  "enchCrack.xpSeed.many": "Liczba możliwych ziarn XP: %d",
+  "enchCrack.playerRNGSeed.many": "Liczba możliwych RNG gracza: %d",
+  "enchCrack.addInfo": "Dodaj Informacje",
+
+  "enchCrack.insn.dummy": "Zaklnij zbędny przedmiot",
+  "enchCrack.insn.ready": "Twoje ziarno zaklęcia jest gotowe",
+  "enchCrack.insn.bookshelves": "Półki na książki: %d",
+  "enchCrack.insn.slot": "W Slocie: %d",
+  "enchCrack.insn.itemThrows": "Ile musisz wyrzucić przedmiotów: %d (about %f seconds)",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/pl_pl.json]: Could `about %f seconds` also be translated?

File: src/main/resources/assets/clientcommands/lang/pl_pl.json [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  "enchCrack.insn.itemThrows.noDummy": "Nie potrzebujesz zbędnego przedmiotu",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/pl_pl.json:81]: Google translate tells me that "zbędnego" means "unnecessary". The English is more trying to hint at a "throwaway" item more than an unnecessary one. Putting "throwaway" into Google translate and looking at the adjective, it's giving me "do jednorazowego użytku". Would that be more suitable?

Comment (L0um15) [reply src/main/resources/assets/clientcommands/lang/pl_pl.json:81]: Polish language is very difficult to translate and as i said we have many meanings. "do jednorazowego użytku" means "one time use" and "zbędnego" means that you're not gonna use that enchanted item and wanna disenchant that in order to use it again as a dummy. Im not saying that "do jednorazowego użytku" does not fit at all. But i think that "zbędnego" fits perfectly. Short Answer: Both are correct.

File: src/main/resources/assets/clientcommands/lang/pl_pl.json (hunk 1 above)
Comment (L0um15) [reply src/main/resources/assets/clientcommands/lang/pl_pl.json]: I didn't saw that. I edited it now.

File: src/main/resources/assets/clientcommands/lang/pl_pl.json (hunk 2 above)
Comment (L0um15) [reply src/main/resources/assets/clientcommands/lang/pl_pl.json:81]: Zbędny also means something with no value as wooden_sword

Comment (Earthcomputer) [reply src/main/resources/assets/clientcommands/lang/pl_pl.json:81]: I'll take your word for it, just checking

**Review by Earthcomputer — APPROVED** [review]
Ty!

**Pull request merged**